CommonTools/CandAlgos/plugins/CandPtrProjector.cc

```diff
@@ -16,11 +17,14 @@ class CandPtrProjector : public edm::global::EDProducer<> {
 private:
   edm::EDGetTokenT<edm::View<reco::Candidate>> candSrcToken_;
   edm::EDGetTokenT<edm::View<reco::Candidate>> vetoSrcToken_;
+  bool useDeltaRforFootprint_;
 };
 
 CandPtrProjector::CandPtrProjector(edm::ParameterSet const& iConfig)
     : candSrcToken_{consumes<edm::View<reco::Candidate>>(iConfig.getParameter<edm::InputTag>("src"))},
       vetoSrcToken_{consumes<edm::View<reco::Candidate>>(iConfig.getParameter<edm::InputTag>("veto"))} {
+  useDeltaRforFootprint_ =
+      iConfig.exists("useDeltaRforFootprint") ? iConfig.getParameter<bool>("useDeltaRforFootprint") : false;
```

Comment:
I suppose that this config parameter is still useful/needed here (regardless of the PUPPI MET developments).
Please add this as a required parameter.